stetho/src/main/java/com/facebook/stetho/inspector/protocol/module/DOM.java

```diff
@@ -172,64 +212,245 @@ public void run() {
     });
   }
 
-  private Node createNodeForElement(Object element) {
-    mDOMProvider.verifyThreadAccess();
-
-    NodeDescriptor descriptor = mDOMProvider.getNodeDescriptor(element);
-
-    Node node = new Node();
-    node.nodeId = mObjectIdMapper.putObject(element);
-    node.nodeType = descriptor.getNodeType(element);
-    node.nodeName = descriptor.getNodeName(element);
-    node.localName = descriptor.getLocalName(element);
-    node.nodeValue = descriptor.getNodeValue(element);
+  private void updateTree() {
+    //
```

Comment:
Oh, I was just marking code I still wanted to delete / refactor. Like I mentioned before, there's still a small amount of "todo" left in this PR.